tests/.gitignore

```diff
@@ -0,0 +1,3 @@
+
+# Ignore Vagrant tmp files
+vagrant/.vagrant
```

Comment:
This should be in the root .gitignore

Reply:
I disagree. I think it should be in the most local file. We are not expecting vagrant files elsewhere, are we?

Reply:
Okey, but then we should move the following ones from the root `.gitignore` to here:

```
tests/.cache/
tests/broken_update.ext4
tests/core-image-full-cmdline-vexpress-qemu-broken-network.ext4
tests/core-image-full-cmdline-vexpress-qemu.ext4
tests/core-image-full-cmdline-qemux86-64-broken-network.ext4
tests/core-image-full-cmdline-qemux86-64.ext4
tests/mender-image-full-cmdline-rofs-qemux86-64.ext4
tests/mender-image-full-cmdline-rofs-vexpress-qemu.ext4
tests/large_image.dat
tests/results.xml
tests/downloaded-tools
```